### ISSUE TITLE
Pinning operation fails if maven_install.json is empty

### ISSUE THREAD

**Issue by caseyduquettesc** [bazelbuild/rules_jvm_external#860]
```
/external/maven/defs.bzl:3:28: expected an indented block
```

Where `/external/maven/defs.bzl` is:
```
load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_file")
load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
def pinned_maven_install():
```

**Comment by caseyduquettesc** [comment]
Might be related to https://github.com/bazelbuild/rules_jvm_external/pull/818

**Comment by caseyduquettesc** [comment]
False alarm

**Comment by caseyduquettesc** [comment]
I think that https://github.com/bazelbuild/rules_jvm_external/pull/822/files#diff-680ed1b34ec806ef9fcf63148507fe91922eecad5fde47db94ee8efe01f7295fL1222 is the culprit. It doesn't append anything after `pinned_maven_install()`

**Comment by caseyduquettesc** [comment]
Workaround is commenting out the load statement until `maven_install.json` exists, but I think we may at least want a `pass` at the end of the function so its valid.

```
#load("@maven//:defs.bzl", "pinned_maven_install")
#
#pinned_maven_install()
```

```py
http_files = [
    "load(\"@bazel_tools//tools/build_defs/repo:http.bzl\", \"http_file\")",
    "load(\"@bazel_tools//tools/build_defs/repo:utils.bzl\", \"maybe\")",
    "def pinned_maven_install():",
    "    pass"
]
```
